src/types/api/resources/DisputesResource.ts

```diff
@@ -13,6 +13,10 @@ export interface paths {
         /** @description Add @Operation annotation to provide a description */
         get: operations['getDisputeDetail'];
     };
+    '/v1/disputes/{disputePspReference}/accept': {
+        /** @description Add @Operation annotation to provide a description */
+        get: operations['getDisputeDetail'];
```

Comment:
The operation here should be `'acceptDispute'`.
```suggestion
        get: operations['acceptDispute'];
```